src/main/java/calculator/StringOperator.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.function.BinaryOperator;
+
+enum StringOperator {
+    ADD("+", (target, source) -> target + source),
+    SUB("-", (target, source) -> target - source),
+    DIV("/", (target, source) -> target / source),
+    MUL("*", (target, source) -> target * source);
+
+    final private String operator;
+    final private BinaryOperator<Integer> binaryOperator;
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers) 확인 부탁드려요 
```suggestion
    private final String operator;
    private final BinaryOperator<Integer> binaryOperator;
```